Description: Keep only dep variants of current (packaged) versions
Last-Update: 2022-11-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -67,15 +67,11 @@
 array-impls = ["tokio-postgres/array-impls"]
 with-bit-vec-0_6 = ["tokio-postgres/with-bit-vec-0_6"]
 with-chrono-0_4 = ["tokio-postgres/with-chrono-0_4"]
-with-eui48-0_4 = ["tokio-postgres/with-eui48-0_4"]
 with-eui48-1 = ["tokio-postgres/with-eui48-1"]
-with-geo-types-0_6 = ["tokio-postgres/with-geo-types-0_6"]
 with-geo-types-0_7 = ["tokio-postgres/with-geo-types-0_7"]
 with-serde_json-1 = ["tokio-postgres/with-serde_json-1"]
 with-smol_str-01 = ["tokio-postgres/with-smol_str-01"]
-with-time-0_2 = ["tokio-postgres/with-time-0_2"]
 with-time-0_3 = ["tokio-postgres/with-time-0_3"]
-with-uuid-0_8 = ["tokio-postgres/with-uuid-0_8"]
 with-uuid-1 = ["tokio-postgres/with-uuid-1"]
 
 [badges.circle-ci]
